src/frontend/apps/standalone_site/src/features/ClassRoom/components/Update/ClassRoomUpdate.tsx

```diff
@@ -0,0 +1,103 @@
+import { Box } from 'grommet';
+import { DashboardClassroom } from 'lib-classroom';
+import {
+  AppConfig,
+  AppConfigProvider,
+  CurrentResourceContextProvider,
+  ResourceContext,
+} from 'lib-components';
+import { Fragment } from 'react';
+import { useParams } from 'react-router-dom';
+import styled from 'styled-components';
+
+import bbbBackground from 'assets/img/bbbBackground.png';
+import bbbLogo from 'assets/img/bbbLogo.png';
```

Comment:
Can you add in the commit message an explanation why we have copy these two images please

Reply:
I added more explanation on this commit: ✨(website) update classroom